pkg/cfaws/assumer_aws_sso.go

```diff
@@ -214,7 +213,7 @@ func SSODeviceCodeFlowFromStartUrl(ctx context.Context, cfg aws.Config, startUrl
 		}
 	}
 
-	clio.Info("Awaiting authentication in the browser...")
+	clio.Info("Awaiting browser authentication, this may take up to 2 minutes...")
```

Comment:
Thanks @grantjoy 
What do you think about describing the steps a little more?

```suggestion
	clio.Info("Awaiting AWS authentication in the browser")
	clio.Info("You will be prompted to authenticate with AWS in the browser, then you will be prompted to 'Allow'")
	clio.Info("Once you press 'Allow', the sign in process will complete in a few seconds")
```

Reply:
Great idea! Committed.